api/src/domain/updateSidebarCards.ts

```diff
@@ -51,7 +51,11 @@ export const updateSidebarCards: UpdateSidebarCards = (userData: UserData): User
     hideCard("task-progress");
   }
 
-  if (operatingPhase === "UP_AND_RUNNING_OWNING" || operatingPhase === "GUEST_MODE_OWNING") {
+  if (
+    operatingPhase === "UP_AND_RUNNING_OWNING" ||
+    operatingPhase === "GUEST_MODE_OWNING" ||
+    operatingPhase === "UP_AND_RUNNING"
+  ) {
```

Comment:
[boulder] Can we use the `operatingPhases.ts` to control whether this shows or not, similar to the pattern for other UI components?

Reply:
Can you make sure what I did was what you wanted..

Reply:
Yep, that's what I was talking about